log.go

```diff
@@ -40,10 +43,50 @@ func logOutput() (logOutput io.Writer, err error) {
 					}
 					os.Stderr.WriteString(fmt.Sprint(scanner.Text() + "\n"))
 				}
+				if err := scanner.Err(); err != nil {
```

Comment:
I think this check should be move in the for loop to avoid the deadlock; Is that right ?

Reply:
@azr nope. The loop ends when the error is generated, just without sending an EOF to the writer. The deadlock was because I wasn't calling `w.Close()` earlier.

Reply:
ah got it ! I thought it was the loop 🙂